consensus-types/blocks/rodatacolumn_test.go

```diff
@@ -118,8 +119,52 @@ func TestNewRODataColumnWithAndWithoutRoot(t *testing.T) {
 
 func TestDataColumn_BlockRoot(t *testing.T) {
 	root := [fieldparams.RootLength]byte{1}
+	dataColumn := &RODataColumn{root: root}
+	assert.Equal(t, root, dataColumn.BlockRoot())
+}
+
+func TestDataColumn_Slot(t *testing.T) {
+	slot := primitives.Slot(1)
+
 	dataColumn := &RODataColumn{
-		root: root,
+		DataColumnSidecar: &ethpb.DataColumnSidecar{
+			SignedBlockHeader: &ethpb.SignedBeaconBlockHeader{
+				Header: &ethpb.BeaconBlockHeader{
+					Slot: slot,
+				},
+			},
+		},
 	}
-	assert.Equal(t, root, dataColumn.BlockRoot())
+
+	assert.Equal(t, slot, dataColumn.Slot())
+}
+
+func TestDataColumn_ParentRoot(t *testing.T) {
```

Comment:
Same feedback on this test file, we could declare this test package as `blocks_test` which imports the `blocks` package

Reply:
Added a new item in the peerDAS TODO list.
Not doing it in this present PR, since it would impact a lot of non peerDAS specific blocks and blobs code as well.